tests/Wallabag/ImportBundle/Controller/PocketCsvControllerTest.php

```diff
@@ -0,0 +1,155 @@
+<?php
+
+namespace Tests\Wallabag\ImportBundle\Controller;
+
+use Craue\ConfigBundle\Util\Config;
+use Doctrine\ORM\EntityManagerInterface;
+use Predis\Client;
+use Symfony\Component\HttpFoundation\File\UploadedFile;
+use Tests\Wallabag\CoreBundle\WallabagCoreTestCase;
+use Wallabag\CoreBundle\Entity\Entry;
+
+class PocketCsvControllerTest extends WallabagCoreTestCase
+{
+    public function testImportPocketCsv()
+    {
+        $this->logInAs('admin');
+        $client = $this->getTestClient();
+
+        $crawler = $client->request('GET', '/import/pocket_csv');
+
+        $this->assertSame(200, $client->getResponse()->getStatusCode());
+        $this->assertSame(1, $crawler->filter('form[name=upload_import_file] > button[type=submit]')->count());
+        $this->assertSame(1, $crawler->filter('input[type=file]')->count());
+    }
+
+    public function testImportPocketCsvWithRabbitEnabled()
+    {
+        $this->logInAs('admin');
+        $client = $this->getTestClient();
+
+        $client->getContainer()->get(Config::class)->set('import_with_rabbitmq', 1);
+
+        $crawler = $client->request('GET', '/import/pocket_csv');
+
+        $this->assertSame(200, $client->getResponse()->getStatusCode());
+        $this->assertSame(1, $crawler->filter('form[name=upload_import_file] > button[type=submit]')->count());
+        $this->assertSame(1, $crawler->filter('input[type=file]')->count());
+
+        $client->getContainer()->get(Config::class)->set('import_with_rabbitmq', 0);
+    }
+
+    public function testImportPocketCsvBadFile()
+    {
+        $this->logInAs('admin');
+        $client = $this->getTestClient();
+
+        $crawler = $client->request('GET', '/import/pocket_csv');
+        $form = $crawler->filter('form[name=upload_import_file] > button[type=submit]')->form();
+
+        $data = [
+            'upload_import_file[file]' => '',
```

Comment:
@nicosomb what is the difference between this test and the empty one?

Reply:
You're right. I think it's a bad copy/paste action ...

Reply:
Let's handle this later